### PR TITLE
LW-10279 Merge conway-era into master

### PR DESCRIPTION
**Pull request by mirceahasegan**
# Context

In preparation for the Chang HF, `conway-era` will merge to `master`.

- [x] DBSync [13.3.0.0](https://github.com/IntersectMBO/cardano-db-sync/releases/tag/13.3.0.0) is currently a `Pre-release`. Wait for the final one before merging.
- [x] regenerate fixtures
- [x] investigate `long-running` e2e test failures
- [x] investigate `test:ws` env settings

# Proposed Solution

# Important Changes Introduced

### CODE REVIEW

**Comment by mirceahasegan** [comment]
Converted this to draft until the checklist in the description is all complete

**Review by iccicci — APPROVED** [review]
💪 🚀

**Review by iccicci — DISMISSED** [review]

File: .github/workflows/continuous-integration-e2e.yaml [hunk 1]
```diff
@@ -31,7 +31,7 @@ env:
 on:
   pull_request:
   push:
-    branches: ['master']
+    branches: ['master', 'conway-era']
```
Comment (mkazlauskas) [inline .github/workflows/continuous-integration-e2e.yaml:34]:
```suggestion
    branches: ['master']
```

File: .github/workflows/continuous-integration-unit-tests.yaml [hunk 2]
```diff
@@ -7,7 +7,7 @@ env:
 on:
   pull_request:
   push:
-    branches: ['master']
+    branches: ['master', 'conway-era']
```
Comment (mkazlauskas) [inline .github/workflows/continuous-integration-unit-tests.yaml:10]:
```suggestion
    branches: ['master']
```

**Review by mkazlauskas — CHANGES_REQUESTED** [review]

File: .github/workflows/continuous-integration-e2e.yaml (hunk 1 above)
Comment (mirceahasegan) [reply .github/workflows/continuous-integration-e2e.yaml]: done: 4459ae5a468

File: .github/workflows/continuous-integration-unit-tests.yaml (hunk 2 above)
Comment (mirceahasegan) [reply .github/workflows/continuous-integration-unit-tests.yaml]: done: 4459ae5a468

**Review by mkazlauskas — APPROVED** [review]
The day has come! :sunglasses: :tada:

**Review by iccicci — APPROVED** [review]

**Pull request merged**